src/core/events/reporter/utils.test.ts

```diff
@@ -0,0 +1,74 @@
+import { MockEnvironment } from '../../../mock';
+import { NodeCrypto } from '../../../node';
+import { normalizeSuperJsonDocument } from '../../../schema-tools';
+import { anonymizeSuperJson, hashSuperJson } from './utils';
+
+const environment = new MockEnvironment();
+
+describe('MetricReporter utils', () => {
+  // TODO: Proper tests for config hash and anonymization
```

Comment:
Do you want to do that now?

Reply:
not really, it's been there a long time :sweat_smile:

Reply:
I guess it can wait little bit more😅